opus/import/do_import.py

```diff
@@ -431,12 +431,10 @@ def update_mult_table(table_name, field_name, table_column, val, label,
     if disp_order is None:
         # No disp_order specified, so make one up
         # Update the display_order
-        form_type = table_column['pi_form_type']
-        range_func = None
-        if form_type is not None and form_type.startswith('GROUP:'):
-            range_func_name = form_type.replace('GROUP:', '')
-            if range_func_name in opus_support.RANGE_FUNCTIONS:
-                range_func = opus_support.RANGE_FUNCTIONS[range_func_name][1]
+        (form_type, form_type_format,
+         form_type_unit_id) = opus_support.parse_form_type(
```

Comment:
You removed the `import opus_support` so you definitely shouldn't reference `opus_support` like this. I'm actually surprised this works. I would do what the other files do. See, for example, the top of `apps/search/views.py`. Then remove the `opus_support.` entirely.